products/ubuntu.md

```diff
@@ -392,17 +392,20 @@ Ubuntu Pro offers security fixes for critical, high, and selected medium CVEs in
 `universe` repositories. Ubuntu Pro (Infra-only)[^1] only guarantees security fixes for packages in
 the `main` repository.
 
+Canonical also offers [Ubuntu Legacy Support](https://ubuntu.com//blog/canonical-expands-long-term-support-to-12-years-starting-with-ubuntu-14-04-lts),
+to extend the support of Ubuntu 14.04 by another 2 years to April 2026.
```

Comment:
```suggestion
to extend the support of Ubuntu LTS releases from 14.04 by another 2 years beyond Extended Security Maintenance (ESM).
```

WDYT ? Should we update `extendedSupport` accordingly ?

Reply:
The table heading won't match (Ubuntu Pro vs Legacy Support).